hmt_escrow/job.py

```diff
@@ -243,8 +223,8 @@ def __init__(self,
         True
         >>> job.setup()
         True
-        >>> job.launcher()
-        "0x1413862C2B7054CDbfdc181B83962CB0FC11fD92"
```

Comment:
We do want to check that the launcher is the same that launched the contract. I would not remove or modify the existing test.

Reply:
It was getting a different address than `0x1413862C2B7054CDbfdc181B83962CB0FC11fD92`. Is it supposed to be deterministic? If not maybe we can get the event from the factory.

Reply:
I don't think deterministic .. @rbval ?

Reply:
Oh, nvm, ur comment makes sense